climada/hazard/relative_cropyield.py

```diff
@@ -920,20 +889,68 @@ def read_wheat_mask_isimip3(input_dir=None, filename=None, bbox=None):
     combine_crops is True.
 
     Optional Parameters:
-        input_dir (str or Path): path to directory containing input file
+        input_dir (Path or str): path to directory containing input file
         filename (str): name of file
         bbox (tuple): geogr. bounding box, tuple or array with for elements.
 
     Returns:
         whe_mask (xarray)"""
 
     if input_dir is None:
-        input_dir = INPUT_DIR
+        input_dir = Path(INPUT_DIR)
     if filename is None:
-        filename= 'winter_and_spring_wheat_areas_phase3.nc4'
+        filename = 'winter_and_spring_wheat_areas_phase3.nc4'
     if bbox is None:
         bbox = BBOX
 
     whe_mask = xr.open_dataset(Path(input_dir, filename), decode_times=False)
     [lonmin, latmin, lonmax, latmax] = bbox
     return whe_mask.sel(lon=slice(lonmin, lonmax), lat=slice(latmax, latmin))
+
+def plot_comparing_maps(haz_his, haz_fut, axes, nr_cli_models=1, model=1):
+    """Plots comparison maps of historic and future data and their difference fut-his
+
+    Parameters:
+        haz_his (RelativeCropyield): historic hazard
+        haz_fut (RelativeCropyield): future hazard
+        axes (Geoaxes): subplot axes that can be generated with ag_drought_util.setup_subplots
+        nr_cli_models (int): number of climate models and respectively nr of rows within
+                                the subplot
+        model (int): current row to plot - this method can be used in a loop to plot
+            subplots in one figure consisting of several rows of subplots.
+            One row displays the intensity for present and future climate and the difference of
+            the two for one model-combination (ag_model and cl_model)
+
+    Returns:
+        geoaxes
```

Comment:
Use NumPy docstring format. I know, it's only a copy-paste, but maybe you can take the opportunity :+1: